### ISSUE TITLE
Defining a dev container which doesn't exist causes a nil pointer dereference

### ISSUE THREAD

**Issue by phoenix-bjoern** [okteto/okteto#3834]
When defining a dev container in the manifest which doesn't exist in the deployment causes an error:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb8 pc=0x1015c4d79]

goroutine 103 [running]:
github.com/okteto/okteto/pkg/k8s/apps.ValidateMountPaths(0xc001866680?, 0xc001b9c600)
        github.com/okteto/okteto/pkg/k8s/apps/validation.go:29 +0x99
github.com/okteto/okteto/cmd/up.(*upContext).activate(0xc001bb89a0)
        github.com/okteto/okteto/cmd/up/activate.go:95 +0x5b7
github.com/okteto/okteto/cmd/up.(*upContext).activateLoop(0xc001bb89a0)
        github.com/okteto/okteto/cmd/up/up.go:705 +0x296
created by github.com/okteto/okteto/cmd/up.(*upContext).start
        github.com/okteto/okteto/cmd/up/up.go:637 +0x6e5
```

Manifest:

```
dev:
  mydeployment:
    command: "bash"
    container: okteto
```

Tested with okteto version 2.18.0.

**Comment by phoenix-bjoern** [comment]
This issue is still open and shouldn't be closed.

**Comment by phoenix-bjoern** [comment]
This issue is still open and shouldn't be closed.

**Comment by phoenix-bjoern** [comment]
This issue is still open and shouldn't be closed.

**Comment by Bharadwajshivam28** [comment]
Hey @phoenix-bjoern I can take this issue and work on it

**Comment by phoenix-bjoern** [comment]
@Bharadwajshivam28 thanks for getting back to this. the issue has been already described in my first comment: In Kubernetes you've a deployment "myapp". But in the Okteto manifest (`okteto.yaml`) you're referencing `dev.mydeployment`. This miss-match caused a nil-pointer dereference exception in Okteto CLI v2.18.0. I just tried to reproduced it with Okteto CLI v2.24.2 but it seems to be solved. Closing the ticket.
